domains/runtime/auto-id/src/lib.rs

```diff
@@ -872,6 +873,33 @@ impl_runtime_apis! {
             }
         }
 
+        fn extract_signer_if_all_within_tx_range(
+            extrinsics: &Vec<<Block as BlockT>::Extrinsic>,
```

Comment:
This would be more readable as `Vec<ExtrinsicFor<Block>>`, maybe we could do a search and replace as a follow-up?
(We already have that type alias, but it would need to move to `subspace_runtime_primitives`.)

Reply:
Done https://github.com/autonomys/subspace/pull/3531, depends on which PR merged first the second PR won't be compiled since `Block as BlockT` is removed, and I will fix the conflict by then.